src/lib/icons.tsx

```diff
@@ -37,8 +39,13 @@ async function getIcons(dirName: string): Promise<Icon[]> {
   const fileNames = await fs.readdir(dirPath);
   return await Promise.all(
     fileNames.map(async (fileName) => {
+      const id = fileName.replace(/\.[^/.]+$/, '');
+
+      const md = metaData.find((m) => m.id === id);
```

Comment:
use a better var name than `md`

Reply:
What if we use something like `currentFileMetaData` instead of just `md` here?